test/integration/helpers.bash

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+# Root directory of the repository.
+SWARM_ROOT=${BATS_TEST_DIRNAME}/../..
+
+# Docker image and version to use for integration tests.
+DOCKER_IMAGE=${DOCKER_IMAGE:-aluzzardi/docker}
+DOCKER_VERSION=${DOCKER_VERSION:-1.5}
+
+# Host on which the manager will listen to (random port between 6000 and 7000).
+SWARM_HOST=127.0.0.1:$(( ( RANDOM % 1000 )  + 6000 ))
+
+# Use a random base port (for engines) between 5000 and 6000.
+BASE_PORT=$(( ( RANDOM % 1000 )  + 5000 ))
+
+# Join an array with a given separator.
+function join { local IFS="$1"; shift; echo "$*"; }
```

Comment:
This comment is just about consistent style. Should `join` have parentheses after it to be consistent with other function definitions? FWIW if you have the parens, you don't need the `function` keyword.